tests/php/Tasks/MigrateSiteTreeLinkingTaskTest.php

```diff
@@ -12,8 +12,6 @@ class MigrateSiteTreeLinkingTaskTest extends SapphireTest
 {
     protected static $fixture_file = 'MigrateSiteTreeLinkingTaskTest.yml';
 
-    protected static $use_draft_site = true;
```

Comment:
Removed because FunctionalTest::use_draft_site is deprecated. This never actually needed to be set true for the unit test to pass.